src/disrupt.sh

```diff
@@ -31,7 +31,7 @@ if [[ "${DICE_ROLL}" -eq 1 ]]; then
     export INCLUDE_IMAGE=1
 
     # Choose a random image from the image assets directory
-    RANDOM_IMAGE=$(printf '%q' "${IMAGES_LIST}" | shuf -n 1)
+    RANDOM_IMAGE=$(printf '%s' "${IMAGES_LIST}" | shuf -n 1)
```

Comment:
Is there documentation on max length of %s? As long as the unit test names are more than whatever android and facebook files are named usually it should be fine tho